.rubocop.yml

```diff
@@ -15,3 +15,10 @@ inherit_mode:
 #
 # See https://github.com/alphagov/rubocop-govuk/blob/main/CONTRIBUTING.md
 # **************************************************************
+
+Style/MultilineBlockChain:
+  # This is a poorly thought out rule that conflicts with Style/BlockDelimiters.
+  # Style/MultilineBlockChain says use do...end instead of {...} for multi-line blocks,
+  # but Style/BlockDelimiters says use {...} instead of do...end when chaining.
+  # The latter should take priority over the former, because it leads to more readable code.
+  Enabled: false
```

Comment:
This has caught me out a few times too! Is there a case for feeding this back upstream into rubocop-govuk?

Reply:
Good shout! I've raised alphagov/rubocop-govuk#365 to upstream this